engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/MultiPartitionDeploymentLifecycleTest.java

```diff
@@ -213,4 +218,26 @@ public void shouldRejectCompleteDeploymentDistributionWhenAlreadyCompleted() {
         .containsExactlyInAnyOrder(
             RecordType.COMMAND, RecordType.COMMAND, RecordType.COMMAND_REJECTION);
   }
+
+  @Ignore("Regression https://github.com/camunda/zeebe/issues/13233")
```

Comment:
❓ Why do we ignore it though? I am so confused

Reply:
It's just pre-work to resolve https://github.com/camunda/zeebe/issues/13233 eventually. It can be un-ignored when we resolve https://github.com/camunda/zeebe/issues/13233.